### PR TITLE
dev

### PR DESCRIPTION
**Pull request by Tam-Pham**

### CODE REVIEW

File: neurokit2/ecg/ecg_phase.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        A dictionary containing additional information of ecg delineation and
+        can be obtained with `ecg_delineate()`.
+
+    Returns
+    -------
+    signals : DataFrame
+        A DataFrame of same length as `ecg_signal` containing the following
+        columns:
+
+        - *"Atrial_Phase"*: cardiac phase, marked by "1" for systole
```
Comment (DominiqueMakowski) [inline neurokit2/ecg/ecg_phase.py]: Would rename it:
```suggestion
        - *"ECG_Phase_Artrial"*: cardiac phase, marked by "1" for systole
```

File: neurokit2/ecg/ecg_phase.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+import pandas as pd
+
+from ..signal import signal_phase
+
+
+
+def ecg_phase(ecg_cleaned, rpeaks=None, delineate_info=None, method='dwt',  sampling_rate=None):
+    """Compute cardiac phase (for both atrial and ventricular).
+
+    Finds the cardiac phase, labelled as 1 for systole and 0 for diastole.
```
Comment (DominiqueMakowski) [inline neurokit2/ecg/ecg_phase.py]: I'd add a bit of info about what's the difference between these two types of phase and also which one is the "major" one (the most studied in regards to cognition for instance)

File: neurokit2/ecg/ecg_phase.py (hunk 1 above)
Comment (Tam-Pham) [reply neurokit2/ecg/ecg_phase.py]: Agree! It's clearer that way

File: neurokit2/ecg/ecg_phase.py [hunk 3]
```diff
@@ -28,7 +28,7 @@ def ecg_phase(ecg_cleaned, rpeaks=None, delineate_info=None, method='dwt',  samp
         A DataFrame of same length as `ecg_signal` containing the following
         columns:
 
-        - *"Atrial_Phase"*: cardiac phase, marked by "1" for systole
+        - *"ECG_Phase_Artrial"*: cardiac phase, marked by "1" for systole
           and "0" for diastole.
         - *"Ventricular_Phase"*: cardiac phase, marked by "1" for systole
```
Comment (DominiqueMakowski) [inline neurokit2/ecg/ecg_phase.py]: Same applies here, obviously 😁

**Comment by DominiqueMakowski** [comment]
Let's re-run the README + all examples with the newest version before merging

**Pull request merged**